### PR TITLE
remove space edit ng form and replace it with the original one

### PR DESCRIPTION
**Pull request by denisdimitrov**
- Removes `SpaceEditFormNG` and replaces it with the original one, `SpaceEditForm`
- Adds the missing auditorium fields to the original component

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/1531

### CODE REVIEW

**Review by colinhowe — APPROVED** [review]

**Comment by denisdimitrov** [comment]
Do not merge. The validation needs to be refactored after the conflict resolve.

File: src/components/atoms/SpacesDropdown/SpacesDropdown.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const SpacesDropdown: React.FC<SpacesDropdownProps> = ({
 
         return (
           <ReactBootstrapDropdown.Item
-            key={id}
+            key={id + name}
```
Comment (colinhowe) [inline src/components/atoms/SpacesDropdown/SpacesDropdown.tsx:63]: `id` is unique. It shouldn't need name adding to it.

Comment (colinhowe) [inline src/components/atoms/SpacesDropdown/SpacesDropdown.tsx:63]: `id` is unique. It shouldn't need `name` adding to it.
```suggestion
            key={id}
```

**Review by colinhowe — CHANGES_REQUESTED** [review]
One minor change and then this can be merged

File: src/components/atoms/SpacesDropdown/SpacesDropdown.tsx (hunk 1 above)
Comment (denisdimitrov) [reply src/components/atoms/SpacesDropdown/SpacesDropdown.tsx:63]: Looks like it isn't. I don't know why but React warning was showing up for duplicated keys.

**Comment by colinhowe** [comment]
Superseded by https://github.com/sparkletown/sparkle/pull/2658

**Pull request closed without merging**